wfd/public/video_format.h

```diff
@@ -147,6 +150,25 @@ struct H264VideoFormat {
   RateAndResolution rate_resolution;
 };
 
+struct SupportedH264VideoFormats {
```

Comment:
See the commit message:
https://github.com/01org/wysiwidi/commit/3b5390ea714b09bf99752f118c20e049a96a76e7

SupportedFormats is a single profile, level and a *set* of resolutions. That's how sinks communicate which resolutions they support to the sources. SelectableFormat is a single profile, level and only one resolution - that's how sources reply to the sink what they have selected.

On the protocol level, sinks send resolution bitmasks with multiple bits set, while sources reply with resolution bitmasks where only one bit is set.